source/changelog.txt

```diff
@@ -12,6 +12,23 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.6.1
+------
+
+*Released December 1, 2022*
+
+- :issue:`MONGOSH-1320`: Fixes a startup bug related to Docker and 
+  similar environments.
+
+- :issue:`MONGOSH-955`: ``printjson()`` now prints full, untruncated 
+  output.
```

Comment:
We don't document this method, and I wouldn't necessarily include this item here, as I'm not really convinced that it's a good idea to actually advocate its usage (it's called `printjson` and it doesn't print JSON, for starters). It's only present for legacy shell compatibility, and was included in the Slack announcement since some internal users feel strongly about its behavior.

Reply:
Thanks for the explanation. I've removed this item.